### PR TITLE
Operator Rebrand - Import wallet modal

### PR DESCRIPTION
**Pull request by stanyyyyy**
Added to styles to Import wallet modal on sentry wallet page.
[Ticket](https://www.pivotaltracker.com/n/projects/2671427/stories/187796484)

### CODE REVIEW

File: apps/sentry-client-desktop/src/features/home/modals/ImportSentryAlertModal.tsx [hunk 1]
```diff
@@ -9,38 +12,39 @@ interface ImportSentryAlertModalProps {
 export function ImportSentryAlertModal({setShowModal, onSuccess}: ImportSentryAlertModalProps) {
 	return (
 		<div
-			className="fixed top-0 right-0 left-0 bottom-0 m-auto w-auto h-auto flex flex-col justify-start items-center z-30">
-			<div className="w-full h-full bg-white opacity-75"/>
+			className="fixed top-0 right-0 left-0 bottom-0 m-auto w-auto h-auto flex flex-col justify-start items-center z-[60]">
+			<div className="w-full h-full bg-[#2A2828] opacity-75"/>
```
Comment (bsCryptit) [inline apps/sentry-client-desktop/src/features/home/modals/ImportSentryAlertModal.tsx]: Why not use theme colors?

File: apps/sentry-client-desktop/src/features/home/modals/ImportSentryAlertModal.tsx [hunk 2]
```diff
@@ -9,38 +12,39 @@ interface ImportSentryAlertModalProps {
...
 			<div
-				className="absolute top-0 right-0 left-0 bottom-0 m-auto flex flex-col justify-start items-center w-[506px] h-[272px] border border-gray-200 bg-white">
+				className="absolute top-0 right-0 left-0 bottom-0 m-auto flex flex-col justify-start items-center w-[692px] h-[200px] bg-black">
 				<div
-					className="absolute top-0 right-0 h-16 flex flex-row justify-between items-center text-lg px-6">
+					className="absolute top-0 right-0 h-16 flex flex-row justify-between items-center text-lg px-6 mt-[18px]">
 					<div className="cursor-pointer z-10" onClick={() => setShowModal(false)}>
-						<AiOutlineClose/>
+						<AiOutlineClose size={24} color="white" className="hover:!text-[#FF0030] duration-300 ease-in" />
```
Comment (bsCryptit) [inline apps/sentry-client-desktop/src/features/home/modals/ImportSentryAlertModal.tsx]: Why not use theme colors?

Comment (bsCryptit) [reply apps/sentry-client-desktop/src/features/home/modals/ImportSentryAlertModal.tsx:22]: Resolved in Mattermost 👍

File: apps/sentry-client-desktop/src/features/home/modals/ImportSentryAlertModal.tsx (hunk 1 above)
Comment (IhorCryptit) [reply apps/sentry-client-desktop/src/features/home/modals/ImportSentryAlertModal.tsx]: Color changed

**Pull request merged**